src/zcl/definition/cluster.ts

```diff
@@ -2872,6 +2872,12 @@ const Cluster: {
                     {name: 'strobelevel', type: DataType.uint8},
                 ],
             },
+            boschOutdoorSiren: {
+                ID: 0xf0,
+                parameters: [
+                    {name: 'data', type: DataType.uint8},
```

Comment:
```suggestion
                    {name: 'data', type: DataType.uint8, manufacturerCode: ManufacturerCode.Bosch},
```

Can you check if it also works with this?